### PR TITLE
Calculate positions relative to parent node

### PR DESCRIPTION
**Pull request by IamNotUrKitty**
Fix some problems with nested `FlipMove`, reference to [#12](https://github.com/joshwcomeau/react-flip-move/issues/12)

### CODE REVIEW

File: src/FlipMove.js [hunk 1]
```diff
@@ -83,10 +98,14 @@ class FlipMove extends Component {
...
+    const relativeBox = {
+      'top':newBox.top - this.parentBox.top,
+      'left':newBox.left - this.parentBox.left
+    };
 
     return [
-      oldBox.left - newBox.left,
-      oldBox.top  - newBox.top
+      Math.abs(oldBox.left) - Math.abs(relativeBox.left),
+      Math.abs(oldBox.top)  - Math.abs(relativeBox.top)
```
Comment (joshwcomeau) [inline src/FlipMove.js]: Quick question: Why is the `Math.abs` necessary? Seems to me like if the old box was -20px from the parent and the new box is 20px, we want it to move by 40px, not 0px?

Thanks so much for the PR =)

**Comment by IamNotUrKitty** [comment]
Yes you right, it not necessary. Totally forgot for what reason i do it. May be for first i try calculate position something like `parentBox.top - boundingBox.top` and get negative values. Now it not necessary for sure.

**Comment by joshwcomeau** [comment]
Makes sense! 

Everything else looks perfect, and the tests pass. Will merge it in if you can update the PR (If I don't hear from you in a bit, I'll merge it in anyway and fix it up in a rebase. So don't worry if you're busy :))

:rocket:

**Comment by IamNotUrKitty** [comment]
Updated.

**Pull request merged**